data/CA/incentives.json

```diff
@@ -0,0 +1,2858 @@
+[
+  {
+    "id": "CA-1",
+    "authority_type": "utility",
+    "authority": "ca-bear-valley-electric-service",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "ca_bearValleyElectricService_energySavingsAssistanceProgram(ESAP)",
+    "amount": {
+      "type": "percent",
+      "number": 100
+    },
+    "owner_status": [
```

Comment:
I may need to leverage this mapping for homeowner vs renter, but this may require going through each authority and figuring out if they explicitly state homeowner vs tenant  https://github.com/rewiringamerica/incentive-data-llm/blob/main/src/constants.ts#L16